src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -60,6 +60,61 @@ using namespace SCIRun::Render;
 using namespace SCIRun::Render::Gui;
 using namespace SCIRun::Modules::Render;
 
+namespace SCIRun {
+namespace Gui {
+  class SCISHARE ScopedWidgetColorChanger
+  {
+  public:
+    ScopedWidgetColorChanger(WidgetHandle widget)
+    : widget_(widget)
+    {
+      backupColorValues();
```

Comment:
```suggestion
      if (widget_) { backupColorValues(); colorWidgetRed(); }
```

Reply:
Since there will be other colors added when middle and right click are added from the features in the bounding box widget, I put a switch statement to allow green and blue as well.

Reply:
ok